charts/reporter/templates/bootstrap-mongodb.yaml

```
@@ -4,7 +4,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: reporter-bootstrap-mongodb
+  name: {{ include "reporter.name" . }}-bootstrap-mongodb
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)